nixos/tests/all-tests.nix

```
@@ -598,6 +598,7 @@ in {
   lightdm = handleTest ./lightdm.nix {};
   lighttpd = handleTest ./lighttpd.nix {};
   limesurvey = handleTest ./limesurvey.nix {};
+  limine = handleTest ./limine { inherit recurseIntoAttrs runTest; };
```

Very minor nitpick, this seems to be more common in this file:
```suggestion
  limine = pkgs.recurseIntoAttrs (handleTest ./limine {});
```